Sources/sys/Resources.swift

```diff
@@ -108,14 +108,18 @@ public final class Resources {
     public static func findExecutable(name: String) -> String {
         // If we have an executable path, look adjacent to it first.
         if let executablesPath = executablesPath {
-            let p = Path.join(executablesPath, name)
-            if p.exists {
-                return p
+            let path = Path.join(executablesPath, name)
+            if path.exists {
+                return path
             }
         }
 
-        // FIXME: Otherwise, search PATH.
-        
+        // Then try to find it from $PATH.
+        // We may need to use "where.exe" on Windows. Currently only for OS X and Linux.
+        if let path = try? popen(["which", name]) {
```

Comment:
Depending on `which` may be unacceptable. I am sure there are platforms that don't have it available as a shell builtin. Also possibly this will break some things. I'm not sure the function is intended to search all paths. /cc @ddunbar